src/pal/src/misc/sysinfo.cpp

```diff
@@ -94,13 +94,10 @@ SET_DEFAULT_DEBUG_CHANNEL(MISC);
 #endif
 #endif // __APPLE__
 
-/*++
-Function:
-    GetNumberOfProcessors
 
-Return number of processors available for the current process
---*/
-int GetNumberOfProcessors()
+DWORD
+PALAPI
+PAL_GetLogicalCpuCountFromOS()
```

Comment:
This function existed already and is used by GetLogicalCpuCountFromOS in util.cpp.